.devcontainer/dev.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:34
+FROM registry.fedoraproject.org/fedora:36
```

Comment:
Don't know if it's this specifically or something else, but I get a failure trying to build the container image:
```
  Building wheel for cchardet (setup.py): started
  Building wheel for cchardet (setup.py): finished with status 'error'
  ERROR: Command errored out with exit status 1:
   command: /usr/bin/python3 -u -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-62yk93_h/cchardet_c59cf2bd70124c5194847d4a8830353c/setup.py'"'"'; __file__='"'"'/tmp/pip-install-62yk93_h/cchardet_c59cf2bd70124c5194847d4a8830353c/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-ynhhvp0k
       cwd: /tmp/pip-install-62yk93_h/cchardet_c59cf2bd70124c5194847d4a8830353c/
  Complete output (21 lines):
  running bdist_wheel
  running build
  running build_py
  creating build
  creating build/lib.linux-x86_64-3.10
  creating build/lib.linux-x86_64-3.10/cchardet
  copying src/cchardet/__init__.py -> build/lib.linux-x86_64-3.10/cchardet
  copying src/cchardet/version.py -> build/lib.linux-x86_64-3.10/cchardet
  running build_ext
  building 'cchardet._cchardet' extension
  creating build/temp.linux-x86_64-3.10
  creating build/temp.linux-x86_64-3.10/src
  creating build/temp.linux-x86_64-3.10/src/cchardet
  creating build/temp.linux-x86_64-3.10/src/ext
  creating build/temp.linux-x86_64-3.10/src/ext/uchardet
  creating build/temp.linux-x86_64-3.10/src/ext/uchardet/src
  creating build/temp.linux-x86_64-3.10/src/ext/uchardet/src/LangModels
  gcc -Wno-unused-result -Wsign-compare -DDYNAMIC_ANNOTATIONS_ENABLED=1 -DNDEBUG -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -O2 -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fstack-protector-strong -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -D_GNU_SOURCE -fPIC -fwrapv -fPIC -Isrc/ext/uchardet/src -I/usr/include/python3.10 -c src/cchardet/_cchardet.cpp -o build/temp.linux-x86_64-3.10/src/cchardet/_cchardet.o
  gcc: fatal error: cannot execute ‘cc1plus’: execvp: No such file or directory
  compilation terminated.
  error: command '/usr/bin/gcc' failed with exit code 1
```
really haven't the slightest idea what to do here but google :)

Reply:
seems i just [need gcc-c++](https://www.unixmen.com/fix-gcc-error-trying-exec-cc1plus-execvp-file-directory-fedora/)